src/app/shared/services/lesson-teachers-rest.service.ts

```diff
@@ -0,0 +1,31 @@
+import { Inject, Injectable } from '@angular/core';
+import { RestService } from './rest.service';
+import { HttpClient, HttpParams } from '@angular/common/http';
+import { Settings, SETTINGS } from '../../settings';
+import { Observable } from 'rxjs';
+import { switchMap } from 'rxjs/operators';
+import { decodeArray } from '../utils/decode';
+import { LessonAbsence } from '../models/lesson-absence.model';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class LessonTeachersRestService extends RestService<
+  typeof LessonAbsence
+> {
+  constructor(http: HttpClient, @Inject(SETTINGS) settings: Settings) {
+    super(http, settings, LessonAbsence, 'LessonTeachers');
+  }
+
+  loadOtherLessonAbsences(
+    personId: number,
+    params?: HttpParams | Dict<string>
+  ): Observable<ReadonlyArray<LessonAbsence>> {
+    return this.http
+      .get<unknown>(
+        `${this.baseUrl}/except/${personId}/LessonAbsences?expand=LessonRef`,
```

Comment:
Ich finde die Lösung für die Typen so in Ordnung. IMHO hättest du alternativ eine neues Basis-Object wie z.B. `ExtendedLessonAbsence` machen müssen und dies hier zurückgeben. Insofern finde ich die aktuelle Lösung zweckmässiger.